app/assets/javascripts/jquery.textfill.min.js

```diff
@@ -0,0 +1,20 @@
+/*
+  textfill
+ @name      jquery.textfill.js
+ @author    Russ Painter
+ @author    Yu-Jie Lin
+ @author    Alexandre Dantas
+ @version   0.6.0
+ @date      2014-08-19
+ @copyright (c) 2014 Alexandre Dantas
+ @copyright (c) 2012-2013 Yu-Jie Lin
+ @copyright (c) 2009 Russ Painter
+ @license   MIT License
+ @homepage  https://github.com/jquery-textfill/jquery-textfill
+ @example   http://jquery-textfill.github.io/jquery-textfill/index.html
+*/
+(function(m){m.fn.textfill=function(r){function f(){a.debug&&"undefined"!=typeof console&&"undefined"!=typeof console.debug&&console.debug.apply(console,arguments)}function s(){"undefined"!=typeof console&&"undefined"!=typeof console.warn&&console.warn.apply(console,arguments)}function p(a,b,e,k,n,g){function d(a,b){var c=" / ";a>b?c=" > ":a==b&&(c=" = ");return c}f("[TextFill] "+a+" { font-size: "+b.css("font-size")+",Height: "+b.height()+"px "+d(b.height(),e)+e+"px,Width: "+b.width()+d(b.width(),
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Unmatched '{'.<br>Unmatched '('.<br>Wrapping non-IIFE function literals in parens is unnecessary.<br>'a' is not defined.<br>'console' is not defined.